packages/backend/migrations/20240524102509_drop_quote_ratio_columns.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.alterTable('quotes', (table) => {
+    table.string('minExchangeRateNumerator').alter()
+    table.string('minExchangeRateDenominator').alter()
+    table.string('lowEstimatedExchangeRateNumerator').alter()
+    table.string('lowEstimatedExchangeRateDenominator').alter()
+    table.string('highEstimatedExchangeRateNumerator').alter()
+    table.string('highEstimatedExchangeRateDenominator').alter()
+
+    table.dropColumn('additionalFields')
```

Comment:
Originally, `additionalFields` was a temp solution when building out support for multiple payment methods. Now though, since we will not be doing that, I decided to get rid of this extra field.